lib/cli-create-embed-token.ts

```diff
@@ -16,13 +16,17 @@ export default function CliCreateEmbedToken() {
         .option('-k, --accessKey <accessKey>', 'The Power BI workspace collection access key')
         .option('-w, --workspace <workspaceId>', 'The Power BI workspace id')
         .option('-r, --report <reportId>', 'The Power BI report id')
+        .option('-d, --dataset <datasetId>', 'The Power BI dataset id')
+        .option('-s, --scopes [scopes]', 'The permission scopes allowed')
         .option('-u, --username [username]', 'The RLS username')
         .option('--roles [roles]', 'The RLS roles', list);
 
     program.on('--help', function () {
         console.log('  Examples:');
         console.log('');
         console.log('    $ powerbi create-embed-token -c <collection>, -w <workspace> -r <reportId> -k <accessKey> -u [username] --roles [roles1,roles2,...]');
+        console.log('    $ powerbi create-embed-token -c <collection>, -w <workspace> -d <datasetId> -k <accessKey>');
```

Comment:
We should have added scopes for the example to create dataset token as even the basic loading operation requires Dataset.Read permissions